rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3FilterHandler.java

```diff
@@ -126,4 +156,37 @@ private boolean isAcceptedAddress(final InetAddress address) {
         }
     }
 
+    private boolean isValidIpAddress(String ip) {
+        return isIPv4(ip) || isIPv6(ip);
```

Comment:
can we combine `isIPv4` and `isIPv6` methods? Otherwise, we may have to parse inet address twice

Reply:
Done